src/lib/y2storage/clients/inst_disk_proposal.rb

```diff
@@ -88,6 +91,19 @@ def run
       # @return [Integer]
       attr_reader :initial_staging_revision
 
+      # The user has changed partition settings.
+      # Asking if these changes can be overwritten.
+      def overwrite_manual_settings?
+        return true unless @manual_changed
```

Comment:
If you are using `attr_reader :manual_changed`, I would use here the provided method (`manual_changed` instead of `@manual_changed`).